channels/ua.m3u

```diff
@@ -45,13 +35,7 @@ https://stream.uagit.tv/gittv.m3u8
 http://cdn1.live-tv.od.ua:8081/a1od/gtvod-abr/a1od/gtvod-480p/playlist.m3u8
 #EXTINF:-1 tvg-id="GTV.ua" tvg-name="GTV" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="https://i.imgur.com/Rc6UGkb.jpg" group-title="",GTV (720p)
 http://cdn1.live-tv.od.ua:8081/a1od/gtvod-abr/a1od/gtvod-720p/playlist.m3u8
-#EXTINF:-1 tvg-id="HDFashionLifestyle.ua" tvg-name="HD Fashion & Lifestyle" tvg-country="UA" tvg-language="" tvg-logo="" group-title="Lifestyle",HD Fashion & Lifestyle
-http://95.67.47.114/hls/hdfashion_ua_hi/index.m3u8
-#EXTINF:-1 tvg-id="HDFashionLifestyleTV.ua" tvg-name="HD Fashion & Lifestyle TV" tvg-country="UA" tvg-language="" tvg-logo="" group-title="Lifestyle",HD Fashion & Lifestyle TV
-http://95.67.47.114/hls/hdfashion_ua_mid/index.m3u8
```

Comment:
also works, why not to keep it then

Reply:
#EXTINF:-1 tvg-id="HDFashionLifeStyle.ua" tvg-name="HDFashion & LifeStyle" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="https://i.imgur.com/LtQL1q9.jpg" group-title="Lifestyle",HDFashion & LifeStyle
http://95.67.47.115/hls/hdfashion_ua_mid/index.m3u8

Hmm... wonder why they got 2 IPs streaming the same content...

Reply:
Rectified.